qp_shogun/shogun/__init__.py

```diff
@@ -37,6 +37,8 @@
     'Taxonomic Predictions - phylum': 'BIOM',
     'Taxonomic Predictions - genus': 'BIOM',
     'Taxonomic Predictions - species': 'BIOM',
+    'Woltka - gOTU': 'BIOM',
```

Comment:
Per our offline discussion, we think that "per_genome" serves the goal more accurately for the moment.